src/Middleware/RateLimiting/src/RateLimiterServiceCollectionExtensions.cs

```diff
@@ -27,4 +27,18 @@ public static IServiceCollection AddRateLimiter(this IServiceCollection services
         services.Configure(configureOptions);
         return services;
     }
+
+    /// <summary>
+    /// Add rate limiting services and configure the related options.
+    /// </summary>
+    /// <param name="services">The <see cref="IServiceCollection"/> for adding services.</param>
+    /// <returns></returns>
+    public static IServiceCollection AddRateLimiter(this IServiceCollection services)
```

Comment:
Update the other overload to call this one to reduce redundancy.

Please add/update at least one test to call the new overload.

Reply:
Test usage added, it little bit naive, but I'm not sure about expected behaviour in this scenario.